src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -290,6 +304,10 @@ public Command sysIdDynamic(SysIdRoutine.Direction direction) {
         return m_sysIdRoutineToApply.dynamic(direction);
     }
 
+    public void addVisionMeasurement(EstimatedRobotPose pose) {
+        addVisionMeasurement(pose.estimatedPose.toPose2d(), pose.timestampSeconds);
+    }
+
```

Comment:
organization. why is this under `// SYSID` :sob: put it back to a similar place to where you found it